### PR TITLE
DOC-4243: Document the new feature Edit activities and comments

### PR DESCRIPTION
**Pull request by mzorai**

### CODE REVIEW

File: docs/GettingStarted.rst [hunk 1]
```diff
@@ -1340,7 +1343,88 @@ manager.
...
-    .. note:: As an eXo Platform user, you can only delete your own activities. If you are manager of a space, you can delete any activities related to this space.
+.. note:: As an eXo Platform user, you can only delete your own activities. If you are manager of a space, you can delete any activities related to this space.
+
+.. _Editing-Activities-Comments:
+
+Editing activities/comments
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+
+Starting from eXo Platform 5.2, you are able to edit an activity you posted or a comment you wrote.
```
Comment (plamarque) [inline docs/GettingStarted.rst]:
```suggestion
Starting from eXo Platform 5.2, you can edit an activity you posted or a comment you wrote.
```

File: docs/GettingStarted.rst [hunk 2]
```diff
@@ -1340,7 +1343,88 @@ manager.
...
+ 
+.. _Edit_Activity:
+ 
+Editing an activity
+--------------------
+ 
+1. Click the button |image135| on the top right of your activity. Two buttons appears: Edit and Delete.
```
Comment (plamarque) [inline docs/GettingStarted.rst]:
```suggestion
1. Click the pulldown menu on the top right of your activity : |image135|. Two entries appear: Edit and Delete.
```

File: docs/GettingStarted.rst [hunk 3]
```diff
@@ -1340,7 +1343,88 @@ manager.
...
+	
+	|image136|
+
+2. Click on ``Edit`` button --> Your activity text appears in the editor area allowing you to make changes.
```
Comment (plamarque) [inline docs/GettingStarted.rst]:
```suggestion
2. Click on ``Edit`` --> Your activity’s text appears in the editor area allowing you to make changes.
```

File: docs/GettingStarted.rst [hunk 4]
```diff
@@ -1340,7 +1343,88 @@ manager.
...
+
+    |image137|
+
+3. Make the needed changes and then click on ``Update`` button.
+
+.. note:: The ``Update`` button still in disabled status until at leat a change is done.
```
Comment (plamarque) [inline docs/GettingStarted.rst]:
```suggestion
.. note:: The ``Update`` button remains disabled until at leat one change is done.
```

File: docs/GettingStarted.rst [hunk 5]
```diff
@@ -1340,7 +1343,88 @@ manager.
...
+
+4. If you click the ``Cancel`` button, your changes will be ignored and your activity still the same.
```
Comment (plamarque) [inline docs/GettingStarted.rst]:
```suggestion
4. If you click the ``Cancel`` button, your changes will be ignored.
```

File: docs/GettingStarted.rst [hunk 6]
```diff
@@ -1340,7 +1343,88 @@ manager.
...
+
+.. note:: Edition is only applicable on the written text. Attached images, files or link could not be edited.
```
Comment (plamarque) [inline docs/GettingStarted.rst]:
```suggestion
.. note:: Edition is only possible on written text. Attached images, files or link can’t be edited.
```

File: docs/GettingStarted.rst [hunk 7]
```diff
@@ -1340,7 +1343,88 @@ manager.
...
+After saving the chage you made on your activity, a text appears under your name indicating that an edit has been done:
+
+|image140|           
+          
+.. _Edit_Comment:
+ 
+Editing a comment
+------------------
+
+1. Click on the button |image141| in the right of the comment box. Same as for the activity, two buttons appear:
```
Comment (plamarque) [inline docs/GettingStarted.rst]:
```suggestion
1. Click on the pulldown menu at the right of the comment box : |image141|. Just like for activities, two entries appear:
```

File: docs/GettingStarted.rst [hunk 8]
```diff
@@ -1340,7 +1343,88 @@ manager.
...
+   
+   |image142|
+
+2. Click on ``Edit`` button --> Your comment text appears in the editor area allowing you to edit it.
```
Comment (plamarque) [inline docs/GettingStarted.rst]:
```suggestion
2. Click on ``Edit``  --> Your comment’s text appears in the editor area allowing you to edit it.
```

File: docs/GettingStarted.rst [hunk 9]
```diff
@@ -1340,7 +1343,88 @@ manager.
...
+
+   |image143|
+   
+3. Edit your comment and then click on ``Update`` button.
+
+.. note:: The ``Update`` button still disabled until you change the comment.
```
Comment (plamarque) [inline docs/GettingStarted.rst]:
```suggestion
.. note:: The ``Update`` button remains disabled until you change the comment.
```

File: docs/GettingStarted.rst [hunk 10]
```diff
@@ -1340,7 +1343,88 @@ manager.
...
+
+4. If you click the ``Cancel`` button, your changes will be ignored and your comment still the same.
```
Comment (plamarque) [inline docs/GettingStarted.rst]:
```suggestion
4. If you click the ``Cancel`` button, your changes will be ignored.
```

File: docs/GettingStarted.rst [hunk 11]
```diff
@@ -1340,7 +1343,88 @@ manager.
...
+
+.. note:: Unlike the activity, you can change an attached link/image to your comment.
```
Comment (plamarque) [inline docs/GettingStarted.rst]:
```suggestion
.. note:: Unlike activities, you can change an attached link/image to your comment.
```

File: docs/GettingStarted.rst [hunk 12]
```diff
@@ -1340,7 +1343,88 @@ manager.
...
+
+           |image144|
+           
+Same as for activities, after saving the change you made to your comment, a text appears near your name indicating that
```
Comment (plamarque) [inline docs/GettingStarted.rst]:
```suggestion
Like for activities, after saving the change you made to your comment, a text appears near your name indicating that 
```

File: docs/GettingStarted.rst [hunk 13]
```diff
@@ -1340,7 +1343,88 @@ manager.
...
+           |image144|
+           
+Same as for activities, after saving the change you made to your comment, a text appears near your name indicating that 
+an edit has been done:
+
+|image145| 
+
+.. warning:: Activities and comments generated from other aplications such as:
+             
+				 - Adding topic or reply to a forum discussion
```
Comment (plamarque) [inline docs/GettingStarted.rst]:
```suggestion
				 - Adding a topic or replying to a forum discussion
```

File: docs/GettingStarted.rst [hunk 14]
```diff
@@ -1340,7 +1343,88 @@ manager.
...
+				 - Creating or joining a space
+				 - Updating your profile
+				 - Adding/Editing a wiki page
+             
+             are not editable, you just can delete them when you click on |image146| button.
+             
+             Only activities generated following a document's upload/add in Documents application are editable,
```
Comment (plamarque) [inline docs/GettingStarted.rst]:
```suggestion
             Only activities generated following a document upload in Documents application are editable, 
```

**Review by plamarque — CHANGES_REQUESTED** [review]
Suggested language changes

File: docs/GettingStarted.rst [hunk 15]
```diff
@@ -1331,16 +1334,105 @@ manager.
...
+   |image106|
+
+3. Click on ``Delete`` button --> A confirmation pop up appears.
+
+   |image107|
+
+
+4. Click ``Yes`` button in the confirmation message to accept your deletion.
+
+.. note:: As an eXo Platform user, you can only delete your own activities. If you are manager of a space, you can delete any activities related to this space.
```
Comment (hghorbel) [inline docs/GettingStarted.rst]: It misses a note for deleting my own comment. 
Also, administrator can edit and delete comments.

File: docs/GettingStarted.rst [hunk 16]
```diff
@@ -1331,16 +1334,105 @@ manager.
...
 
-2. Hover your cursor over the activity you want to delete, then click
-|image106| that appears.
+.. note:: The ``Update`` button remains disabled until at least one change is done.
 
-|image107|
+4. If you click the ``Cancel`` button, your changes will be ignored.
 
-3. Click **OK** in the confirmation message to accept your deletion.
+.. note:: Edition is only possible on written text. Attached images, files or link can’t be edited.
```
Comment (hghorbel) [inline docs/GettingStarted.rst]: We can edit some images added by the image in the CKEditor toolbar.

File: docs/GettingStarted.rst [hunk 17]
```diff
@@ -1331,16 +1334,105 @@ manager.
...
+           |image144|
+           
+Like for activities, after saving the change you made to your comment, a text appears near your name indicating that 
+an edit has been done:
+
+|image145| 
+
+.. warning:: Activities and comments generated from other aplications such as:
+             
+				 - Adding a topic or replying to a forum discussion
```
Comment (hghorbel) [inline docs/GettingStarted.rst]: - closing, opening and locking a topic too will engender an automatic message. 
- Adding a poll in a topic 
- Editing an event in the agenda
- Moving a wiki page
- Editing space description
- Connexion with a new user

**Review by hghorbel — APPROVED** [review]

**Review by plamarque — COMMENTED** [review]
I didn't see any info related to permalinks which are now under the timestamp. Did I miss it ?

**Comment by mzorai** [comment]
@plamarque it is in the part of permalinks https://github.com/exoplatform/exo-documentation/pull/28/files#diff-e45a7dd2962b79ea809e99ae5a8c639bL1594

File: docs/GettingStarted.rst (hunk 16 above)
Comment (mzorai) [reply docs/GettingStarted.rst]: Like for comments

File: docs/GettingStarted.rst [hunk 18]
```diff
@@ -1391,7 +1394,9 @@ Editing an activity
 After saving the change you made on your activity, the activity creation timestamp will be updated by a new 
 label under your name indicating the time of the last edit:
 
-|image140|           
+|image140|   
+
+.. note:: An administrator is allowed to edit and delete any activity or comment.
```
Comment (plamarque) [inline docs/GettingStarted.rst]:
```suggestion
.. note:: An administrator can edit and delete activities and comments posted by other users.        
```

File: docs/GettingStarted.rst [hunk 19]
```diff
@@ -1348,7 +1348,8 @@ manager.
 
 4. Click ``Yes`` button in the confirmation message to accept your deletion.
 
-.. note:: As an eXo Platform user, you can only delete your own activities. If you are manager of a space, you can delete any activities related to this space.
+.. note:: As an eXo Platform user, you can only delete your own activities/comments. 
+          If you are manager of a space, you can delete any activities or comments related to this space.
```
Comment (plamarque) [inline docs/GettingStarted.rst]:
```suggestion
          If you are manager of a space, you can delete any activities or comments posted in this space.
```

Comment (plamarque) [reply docs/GettingStarted.rst]: @mzorai @hghorbel  can an space manager edit activities/comments in his space ?

**Review by plamarque — CHANGES_REQUESTED** [review]

File: docs/GettingStarted.rst (hunk 19 above)
Comment (mzorai) [reply docs/GettingStarted.rst]: No the space's manager can just delete activities or comments of other users on his space. He can not edit them.

Comment (mzorai) [reply docs/GettingStarted.rst]: rectified in the note

**Review by plamarque — APPROVED** [review]
after discussion I approve this PR because there is a hickup in the merges previously done on master. Changes remaining in this changeset (e.g images) are required to fix the already deployed doc.

**Pull request merged**